MapboxCoreNavigation.podspec

```diff
@@ -41,7 +41,7 @@ Pod::Spec.new do |s|
   s.module_name = "MapboxCoreNavigation"
 
   s.dependency "MapboxNavigationNative", "~> 111.0"
-  s.dependency "MapboxDirections", "~> 2.6"
+  s.dependency "MapboxDirections", "~> 2.7.0"
```

Comment:
`v2.7.1` will be picked up in this case, Mapbox Directions version that in turn picks up Turf that is up to `v2.5.x`.